### PR TITLE
UI and code refactor

### PR DESCRIPTION
**Pull request by nkkls**
!!! fun stuff

if anything breaks please scream to me in discord

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,91 +6,69 @@
...
-        Some features made by <code>@millabasset</code> on Discord!<br>
-        Background music made by <code>@x2corp</code> on Discord!<br>
-        Spawn sounds come from REx:R on Roblox<br>
-        Some sounds made by <code>@tevv_</code> on Discord!<br>
-        UI, major code refactor, and new features by <code>@Spaxolotl</code> on Discord!
-    </p>
-    <button id="playButton" class="invisible" onclick="loadContent()">Play!</button>
+    <p id="gameTitle">Woah It's Rex</p>
+    <div style="display: flex; gap:3em; align-items: center;">
+        <ul id="credits">
```
Comment (SpiritAxolotl) [inline index.html:43]: at some point let's try to move the credits somewhere else (for when more people inevitably contribute)

File: css/css.css [hunk 2]
```diff
@@ -34,11 +71,35 @@
...
+  color: white;
+  padding: 1em;
+  transition: background-color 0.2s, color 0.2s, border 0.5s;
+}
+.movementButtons>*:hover {
+  background-color:white;
+  color: black;
+}
+
 #inventoryDisplay > * {
```
Comment (SpiritAxolotl) [inline css/css.css:103]: I cannot for the life of me figure out how to make this entire thing be the same width as the other buttons. please fix

**Comment by marbelynrye** [comment]
what

File: index.html [hunk 3]
```diff
@@ -102,17 +75,63 @@
...
-    <div class="col" id="col-3">
-        <div id="musicStuff">
-            <button id="musicButton" class="inventoryButtons" onclick="toggleMusic()">Mute Music</button>
-            <p>Music Volume</p>
-            <label class="slidecontainer">
-                <input type="range" min="0" max="100" value="100" onchange="changeMusicVolume(this.value)" id="musicVolume">
-            </label>
+    <div class="col" id="col-2">
+        <div style="max-width: 100%;">
+            <div class="movementButtons">
```
Comment (marbelynrye) [inline index.html:80]: stop using bootstrap

File: index.html [hunk 4]
```diff
@@ -102,17 +74,63 @@
...
```
Comment (marbelynrye) [inline index.html:79]: git sucks paste this: <div class="movementButtons">
                <button id="a" class="movementButton">🡠</button>
                <button id="s" class="movementButton">🡣</button>
                <button id="w" class="movementButton">🡡</button>
                <button id="d" class="movementButton">🡢</button>
                <button onclick="switchMovementMode()" id="movementMode">⭮</button>
                <div id="teleportContainer">
                    <button onclick="switchDistance('down')">🡰</button>
                    <button onclick="switchDistance('up')">🡲</button>
                    <button onclick="teleport()" id="teleportButton" title="Teleports you to 1000m" style="grid-column: span 2;">
                        <span class="emoji">🟫</span>
                    </button>
                </div>
            </div>

**Comment by marbelynrye** [comment]
git sucks paste this ```<div class="movementButtons">
                <button id="a" class="movementButton">🡠</button>
                <button id="s" class="movementButton">🡣</button>
                <button id="w" class="movementButton">🡡</button>
                <button id="d" class="movementButton">🡢</button>
                <button onclick="switchMovementMode()" id="movementMode">⭮</button>
                <div id="teleportContainer">
                    <button onclick="switchDistance('down')">🡰</button>
                    <button onclick="switchDistance('up')">🡲</button>
                    <button onclick="teleport()" id="teleportButton" title="Teleports you to 1000m" style="grid-column: span 2;">
                        <span class="emoji">🟫</span>
                    </button>
                </div>
            </div>```

**Comment by marbelynrye** [comment]
and this ```.movementButtons {
  max-width: 100%;
  display: flex;
  justify-content: stretch;
  align-items: stretch;
  aspect-ratio: 1/0.1;
}
.movementButtons button {
  font-size: 1.2em;
  margin: 0;
  border: none;
  border-bottom: solid #343434;
  background-color: #161b1d;
  flex-grow: 1;
  color: white;
  /* padding: 1em; */
  display: flex;
  justify-content: center;
  align-items: center;
  transition: background-color 0.2s, color 0.2s, border 0.5s;
}
.movementButtons * {
  height: 100%;
}
.movementButtons button:hover {
  background-color:white;
  color: black;
}

#teleportContainer {
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-template-rows:    1fr 1fr;
}```

File: js/main.js [hunk 5]
```diff
@@ -674,3 +708,46 @@ function logFind(type, x, y, variant, atMined, fromReset) {
...
+    "w": {
+        "two": `<i class="fas fa-angles-up"></i>`,
+        "one": `<i class="fas fa-angle-up"></i>`
+    },
+    "s": {
+        "two": `<i class="fas fa-angles-down"></i>`,
+        "one": `<i class="fas fa-angle-down"></i>`
+    }
+}
+
```
Comment (marbelynrye) [inline js/main.js:731]: get rid of this

**Pull request merged**